root/account/components/ApplicationForm.js

```diff
@@ -73,25 +75,26 @@ class RegisterApplicationForm extends React.Component<Props, State> {
   handleOauthTypeChange: (e: SyntheticEvent<HTMLSelectElement>) => void;
 
   handleOauthTypeChange(e: SyntheticEvent<HTMLSelectElement>) {
-    const currentOauthType = e.currentTarget.value;
+    const selectedOauthType = e.currentTarget.value;
     this.setState(prevState => ({
       form: set(
         oauthTypeFieldLens,
-        (currentOauthType: any),
+        ((selectedOauthType: any): OauthTypeT),
         prevState.form,
       ),
     }));
   }
 
   render() {
     return (
-      <form action="/account/applications/register" method="post">
+      <form method="post">
         <FormRowText
           field={this.state.form.field.name}
           label={addColon(l('Name'))}
```

Comment:
So since this file contains translated strings, it has to be moved to somewhere under root/static/scripts/ per https://github.com/metabrainz/musicbrainz-server/pull/723